workloads/kube-burner/run.sh

```diff
@@ -86,11 +106,25 @@ if [[ ${PPROF_COLLECTION} == "true" ]] ; then
   delete_pprof_secrets
   delete_oldpprof_folder
   get_pprof_secrets
-fi 
-run_workload kube-burner-crd.yaml
+fi
+
+echo "WORKLOAD ${WORKLOAD}"
+if [[ ${WORKLOAD} -eq "concurrent-builds" ]]; then
+   app_array=($APP_LIST)
+   echo "conc builds"
```

Comment:
These two echo statements are unnecessary, (workload name is already printed in L92)